monai/data/test_time_augmentation.py

```diff
@@ -193,16 +194,16 @@ def __call__(
             batch_data[self._pred_key] = self.inferrer_fn(batch_data[self.image_key].to(self.device))
             outs.extend([self.inverter(i)[self._pred_key] for i in decollate_batch(batch_data)])
 
-        output: NdarrayOrTensor = np.stack(outs, 0) if isinstance(outs[0], np.ndarray) else torch.stack(outs, 0)
+        output: NdarrayOrTensor = stack(outs, 0)
 
         if self.return_full_data:
             return output
 
         # calculate metrics
-        output_t: torch.Tensor
-        output_t, *_ = convert_data_type(output, output_type=torch.Tensor)  # type: ignore
-        mode, *_ = convert_to_dst_type(torch.mode(output_t.long(), dim=0).values, output, dtype=torch.int64)
-        mean, *_ = convert_to_dst_type(torch.mean(output_t, dim=0), output)
-        std, *_ = convert_to_dst_type(torch.std(output_t, dim=0), output)
-        vvc: float = (torch.std(output_t) / torch.mean(output_t)).item()
-        return mode, mean, std, vvc
+        _mode = mode(output, dim=0)
+        mean = output.mean(0)
+        std = output.std(0)
+        vvc = output.std() / output.mean()
+        vvc = vvc.item() if isinstance(vvc, torch.Tensor) else vvc
```

Comment:
Hi @rijobro ,

I think numpy array also has the `item()` function?

Others look good to me.

Thanks.

Reply:
thanks, updated.